### PR TITLE
Added Root Mean Square Error code and readme

### PR DESCRIPTION
**Pull request by kitrak-rev**
This PR adds the RMSE error metric to the Error metric folder inside Simple MLSelfImplementedAlgos. 
With Readme documentation and a jupyter notebook with a sample for verification and testing against the inbuilt RMSE function in sklearn. 
This PR is a part of resolving issue,#2

### CODE REVIEW

**Review by diffrxction — APPROVED** [review]
PR approved and merged
